src/qml/qgismobileapp.qml

```diff
@@ -446,7 +446,7 @@ ApplicationWindow {
     /* The map canvas */
     MapCanvas {
       id: mapCanvasMap
-      interactive: !screenLocker.enabled
+      interactive: !dashBoard.opened && !screenLocker.enabled && !welcomeScreen.visible && !qfieldSettings.visible && !cloudPopup.visible && !codeReader.visible
```

Comment:
This will become unmaintainable at some point. I guess you already have something in mind to keep track what might be showing over the main window in the future?

Reply:
@suricactus , what I have in mind is that hopefully by us moving onto actively developed Qt 6, we can report issues and see them fixed eventually. We already filed an issue on taphandler propagation issue. Crossing fingers we won't need this kind of workaround much longer.